docs: add documentation for invalidate endpoint (#3692)
webpack/webpack.js.org: src/content/configuration/dev-server.md

T> If you're having trouble, navigating to the `/webpack-dev-server` route will show where files are served. For example, `http://localhost:9000/webpack-dev-server`.

> T> If you want to recompile the bundle. e.g. after a change to the configuration. Navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`.

Reviewer: ```suggestion
T> If you want to recompile the bundle. e.g. after a change to the configuration, navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`.
```

> T> If you want to recompile the bundle. e.g. after a change to the configuration. Navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`.

Reviewer 2: This could be shorter, clearer. Maybe:
```suggestion
T> To manually trigger a recompilation of the bundle, navigate to the `http://localhost:9000/invalidate` route (URL may differ depending on your configuration) of [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate).
```
Or in another way?

> T> If you want to recompile the bundle. e.g. after a change to the configuration. Navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`. 🔑

Reviewer 3: ```suggestion
T> If you want to recompile the bundle. e.g. after a change to the configuration. Navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`.
```

> T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). For example, `http://localhost:9000/invalidate`.

Reviewer 2: It already looks much better, just thinking if you should clarify the example for people just getting to know webpack like this?
```suggestion
T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on you configuration, URL may look like `http://localhost:9000/invalidate`.
```

> T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on you configuration, URL may look like `http://localhost:9000/invalidate`.

Reviewer 2: ```suggestion
T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration, URL may look like `http://localhost:9000/invalidate`.
```

T> If you're having trouble, navigating to the `/webpack-dev-server` route will show where files are served. For example, `http://localhost:9000/webpack-dev-server`.

> T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](https://webpack.js.org/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). For example, `http://localhost:9000/invalidate`.

Reviewer 3: ```suggestion
T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). For example, `http://localhost:9000/invalidate`.
```
